internal/pkg/plugin/gitlabcedocker/delete_test.go

```diff
@@ -1,27 +1,79 @@
-package gitlabcedocker_test
+package gitlabcedocker
 
 import (
+	"os"
+
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
-
-	"github.com/devstream-io/devstream/internal/pkg/plugin/gitlabcedocker"
 )
 
-var _ = Describe("delete", func() {
-	var options map[string]interface{}
-
-	BeforeEach(func() {
-		options = map[string]interface{}{
-			"hostname":    "gitlab.devstream.io",
-			"gitlab_home": "/srv/gitlab",
-			"ssh_port":    8122,
-			"http_port":   8180,
-			"https_port":  8443,
-		}
-	})
+var _ = Describe("Delete", func() {
+
+	Describe("RemoveDirs func", func() {
+
+		var (
+			errs []error
+			dirs []string
+		)
+
+		BeforeEach(func() {
+			DeferCleanup(func() {
+				dirs = nil
+			})
+		})
+
+		JustBeforeEach(func() {
+			// Remove the directories
+			errs = RemoveDirs(dirs)
+			for _, e := range errs {
+				Expect(e).ToNot(HaveOccurred())
+			}
+		})
+
+		AfterEach(func() {
+			// check directories are removed
+			for _, dir := range dirs {
+				_, err := os.Stat(dir)
+				Expect(os.IsNotExist(err)).To(BeTrue())
+			}
+		})
+
+		Context("when the directories are not exist", func() {
+
+			BeforeEach(func() {
+				dirs = []string{"dir/not/exist/1", "dir/not/exist/2"}
+			})
+
+			It("should return no error", func() {
+
+			})
+		})
+
+		Context("when the directories are exist", func() {
```

Comment:
for `Context("when the directories are exist"`, you can use `When("the directories are exist")`

Reply:
Thank you!